pkgs/development/python-modules/onnx/default.nix

```diff
@@ -18,7 +18,7 @@
 
 buildPythonPackage rec {
   pname = "onnx";
-  version = "1.7.0";
+  version = "1.8.0";
 
   # Due to Protobuf packaging issues this build of Onnx with Python 2 gives
   # errors on import.
```

Comment:
The next line of the comment (which I can't add a comment on directly) should be changed to something like `# Support for Python 2 is deprecated as of Onnx v1.8.` or I guess it could be omitted entirely.

Reply:
I'll apply this

Reply:
actually, it's already applied:
```
[nix-shell:~/.cache/nixpkgs-review/pr-103286]$ nix-build ./nixpkgs/ -A pythonPackages.onnx
error: onnx-1.8.0 not supported for interpreter python2.7
```